### DISCUSSION TITLE
It's possible to disable 2FA/MFA by querying the database?

### DISCUSSION THREAD

**Discussion by tassoman** [misskey-dev/misskey#14519] [Q&A] [answered]
Altough the procedure is clear and well translated... Someone lost his 2FA/MFA OTP without backing up their codes... 🤦‍♂️ 
As administrator, can I alter their profile to disable MFA? Maybe could happen by APIs?

**Comment by sousuke0422** [comment, marked as answer]
I think we need to touch the database directly. However, if it is not possible to confirm the identity of the person for security reasons, I don't think it should be done.

---
データベースを直接触る必要があると思う。ただし、セキュリティー的に確実に本人であると確認できないならやるべきではないと思う。

**Reply by tassoman** [reply]
I agree with you, could be dangerous if profile connected email gets hacked. BTW if administrators have evidence by profile's owner identification, 2FA/MFA should be allowed to be revoked.

私も同意します。プロファイルに関連付けられたメールがハッキングされた場合は危険です。ちなみに、管理者がプロファイルの所有者識別による証拠を持っている場合は、2FA/MFA を取り消すことができるはずです。
